src/extension/options-page/DashboardComponents/WalletContent.tsx

```diff
@@ -99,13 +99,7 @@ export const WalletContent = React.forwardRef<HTMLDivElement, WalletContentProps
     const [walletRename, , openWalletRename] = useModal(DashboardWalletRenameDialog)
     const [walletRedPacket, , openWalletRedPacket] = useModal(DashboardWalletRedPacketDetailDialog)
 
-    const setAsDefault = useSnackbarCallback(
-        () => Services.Plugin.invokePlugin('maskbook.wallet', 'setDefaultWallet', wallet!.address),
```

Comment:
`setDefaultWallet` impl also need deleted.

Reply:
If rename it to`setSelectedWallet` will be better. But from my point of view `setDefaultWallet` is both OK.